### ISSUE TITLE
after after 1 Min  deleted my account / через 1 минуту секунда удалила мою учетную запись /

### ISSUE THREAD

**Issue by Mahnazmohammadii** [shamhi/HamsterKombatBot#233]
I think I have done all the steps correctly
I created the user account and ran Hamster
Then I did all the steps carefully on the Ubuntu virtual server, but finally, in less than a minute, my user account was deleted by Telegram and even my phone number was blocked by Telegram.
Please note that my phone number is real and belongs to me
I live in Iran
I have done all the configuration steps correctly and I have attached the images as well
Please help me
what do I do?
I just want the bot to stay online and the number of clicks etc. is not important to me
It can be in the lowest state

RUSIAN : 

Я думаю, что выполнил все действия правильно
Я создал учетную запись пользователя и запустил Hamster
Затем я тщательно проделал все действия на виртуальном сервере Ubuntu, но, наконец, менее чем за минуту моя учетная запись была удалена Telegram и даже мой номер телефона был заблокирован Telegram.
Обратите внимание, что мой номер телефона настоящий и принадлежит мне.
Я живу в Иране
Я правильно выполнил все шаги по настройке и также прикрепил изображения.
пожалуйста, помогите мне
что мне делать?
Я просто хочу, чтобы бот оставался онлайн и количество кликов и т. д. для меня не важно.
Это может быть в самом низком состоянии

1 SCREEN START BOT 
![1](https://github.com/shamhi/HamsterKombatBot/assets/167020867/f776ddd9-7b2b-4949-ae0f-c77a1ea96536)

2 START SUCSESE 
![2](https://github.com/shamhi/HamsterKombatBot/assets/167020867/ae38f570-c5f9-4794-be2b-84b4acf93dad)

3- DELETED ACCOUNT 
![DELETED ACCOUNT](https://github.com/shamhi/HamsterKombatBot/assets/167020867/2a913b81-68d9-4110-a12a-a92952be6072)

4- BANED NUMMBER 
![BANED NUMBER](https://github.com/shamhi/HamsterKombatBot/assets/167020867/71a5f127-b9db-4e1c-aa40-32c2df6b0bc9)

5- .ENV CONFIG FILE 
![EVN CONFIG](https://github.com/shamhi/HamsterKombatBot/assets/167020867/68a2cb88-1932-42c5-a87f-8f475a1a93ba)

NOTE : MU NUMBER FOR IRAN 
MY UBUNTU SERVER GERMANI
MY API ID HASH FOR GERMANY NUMBER AND GET IN GERMANI 

PLEASE HELP ME 
WHAT SHOILD IDO? 
 ----------------------------

ПРИМЕЧАНИЕ. НОМЕР MU ДЛЯ ИРАНА. 
МОЙ СЕРВЕР UBUNTU НА НЕМЕЦКОМ
МОЙ API ID HASH ДЛЯ ГЕРМАНИИ НОМЕР И ПОЛУЧИТЬ НА НЕМЕЦКОМ 

ПОЖАЛУЙСТА, ПОМОГИТЕ МНЕ 
ЧТО Я ДОЛЖЕН ДЕЛАТЬ?

**Comment by Mahnazmohammadii** [comment]
YOU CAN REFRANCE TO THIS MU PROBLM 

Duplicate of #   #202 AND #197

**Comment by okulov28** [comment]
After creating a telegram account, wait a couple of days.  In the bot settings you should specify an Iranian proxy.

**Comment by ButCheR-PRO** [comment]
After you have created a Telegram account with a new number, do not rush to set up the bot right away. Wait at least a few hours, then enter the game using your referral link, buy a couple of cards, make a few manual taps, then exit. Then create a new session through the bot. After this, it is better to delete the second account in your Telegram. If you need a second account, use cloning the Telegram application.

**Comment by Mahnazmohammadii** [comment]
> After creating a telegram account, wait a couple of days. In the bot settings you should specify an Iranian proxy.

Do you mean that I should use Telegram's mtprotp proxy? Where should I enter it?

**Comment by Mahnazmohammadii** [comment]
> After you have created a Telegram account with a new number, do not rush to set up the bot right away. Wait at least a few hours, then enter the game using your referral link, buy a couple of cards, make a few manual taps, then exit. Then create a new session through the bot. After this, it is better to delete the second account in your Telegram. If you need a second account, use cloning the Telegram application.

I have carefully done all this, whatever you said 
I even used the old account
But there is a problem, according to reference 7 that I have added new information, can you estimate what the problem is?
For a newly created user account, please provide me with a suggested .env file according to this recipe 
What values ​​to fill

**Comment by Mahnazmohammadii** [comment]
new information 

please read #237 

full log 
[log.txt](https://github.com/user-attachments/files/15749494/log.txt)

screenshot 
![ban2](https://github.com/shamhi/HamsterKombatBot/assets/167020867/40ccf069-13cd-4730-b6f3-e86b1360a2cb)

**Comment by ButCheR-PRO** [comment]
> new information
> 
> please read #237
> 
> full log [log.txt](https://github.com/user-attachments/files/15749494/log.txt)

Read this:
1 https://github.com/shamhi/HamsterKombatBot/issues/38
2 https://github.com/shamhi/HamsterKombatBot/issues/58
3 https://github.com/shamhi/HamsterKombatBot/issues/86
4 https://github.com/shamhi/HamsterKombatBot/issues/137

**Comment by ButCheR-PRO** [comment]
Dupe https://github.com/shamhi/HamsterKombatBot/issues/202
